src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Machine {
```

Comment:
It is preferrable to use interface instead of abstract class due to you are not providing any state